pkg/generic/thrift/http.go

```diff
@@ -146,18 +146,14 @@ func (r *ReadHTTPResponse) Read(ctx context.Context, method string, isClient boo
 	if !ok {
 		return nil, perrors.NewProtocolErrorWithMsg("TProtocol should be BinaryProtocol")
 	}
-	sName, err := in.ReadStructBegin()
-	if err != nil {
-		return nil, err
-	}
-	sBeginLen := bthrift.Binary.StructBeginLength(sName)
 	// TODO: support exception field
 	fName, typeId, id, err := in.ReadFieldBegin()
 	if err != nil {
 		return nil, err
 	}
-	fBeginLen := bthrift.Binary.FieldBeginLength(fName, typeId, id)
-	transBuf, err := tProt.ByteBuffer().ReadBinary(dataLen - sBeginLen - fBeginLen)
+	_, _ = fName, typeId // not in use?
```

Comment:
We don't need to call FieldBeginLength since it does nothing, which means fName and typeId are not in use anymore. So I think you can remove this line and modify L150 to `_, _, id, err := in.ReadFieldBegin()`